_sass/includes/_katex.scss

```diff
@@ -0,0 +1,7 @@
+.katex-display {
+  overflow: visible;
+}
+
+.katex {
+  whitespace: wrap;
```

Comment:
`whitespace` doesn't seem to be a correct attribute, I see `white-space` online but no `wrap` associated to it.

Reply:
It does not seem to be doing anything